app/components/announcement_banner/announcement_banner.js

```diff
@@ -95,13 +97,15 @@ export default class AnnouncementBanner extends PureComponent {
             color: bannerTextColor,
         };
 
+        const padding = DeviceTypes.IS_IPHONE_X && isLandscape ? {paddingHorizontal: ViewTypes.IOS_HORIZONTAL_LANDSCAPE} : {paddingHorizontal: 0};
```

Comment:
This line is a bit repetitive to me and will require changes in several place in case it needs to update in the future.  There are only few variations and so, would it make sense to place it somewhere and just import as necessary?

Reply:
Thanks for the suggestion Saturninoabril! I'm looking into reducing this into a method and will update the commit.

Reply:
![Simulator Screen Shot - iPhone X - 2019-07-28 at 14 00 52](https://user-images.githubusercontent.com/38697367/62015061-1f3f2500-b176-11e9-811e-3d082963aace.png)
![Simulator Screen Shot - iPhone X - 2019-07-28 at 14 01 00](https://user-images.githubusercontent.com/38697367/62015062-1f3f2500-b176-11e9-8224-ed4a4373b58d.png)
![Simulator Screen Shot - iPhone X - 2019-07-28 at 14 01 24](https://user-images.githubusercontent.com/38697367/62015063-1f3f2500-b176-11e9-9544-0800681a0f10.png)
![Simulator Screen Shot - iPhone X - 2019-07-28 at 14 03 42](https://user-images.githubusercontent.com/38697367/62015064-1f3f2500-b176-11e9-810d-67976eb7d340.png)
Screens have been updated and the padding functions for the majority of cases have been moved into their own file to be imported.

Reply:
I've seen that the last commit was reverted.  Let me know once ready for review.

Reply:
Hi @saturninoabril , this is ready!

Reply:
These are resolved.